app/src/features/datasets/components/RenderWithHandlebars.tsx

```diff
@@ -0,0 +1,28 @@
+import * as DOMPurify from 'dompurify';
+import 'styles/handlebar.scss';
+import { useHandlebars } from 'utils/handlebarsUtils';
+
+export interface IRenderWithHandlebarsProps {
+  datasetEML: {
+    data: {
+      'eml:eml': {
+        dataset: any;
+        additionalMetadata: any;
+      };
```

Comment:
We'll have to be a bit careful in general as we move forward, since we can't presume what the EML will contain, with the exception of any fields that are required by the EML spec of course.